boot/freeldr/freeldr/ntldr/winldr.c

```diff
@@ -1278,6 +1278,16 @@ LoadAndBootWindowsCommon(
     WinLdrpDumpArcDisks(LoaderBlockVA);
 #endif
 
+    /* Show a notice to user about debugger connection */
+    if (strstr(LoaderBlockVA->LoadOptions, "DEBUG"))
+    {
+        MachVideoPuts("Debug mode is enabled.\n"
```

Comment:
We cannot use `TuiPrintf` here instead?
If so, this would allow removing this MachVideoPuts.

Reply:
Nope, it uses `MachConsPutChar` which uses BIOS interrupts which are not available here.

Reply:
ok thanks

Reply:
Yes, it can be anywhere between the MachPrepareForReactOS and the actual call into the kernel.

Reply:
then i would suggest you move it as suggested, this would avoid complicating the PR with introducing this new function.

Reply:
Ok so seems like PrepareForReactOS function doesn't disable BIOS interrupts on PC, but it exits boot services on UEFI, but then UefiConsPutChar uses UefiVideoPutChar in the end to print a character... so this should be ok.